modules/configuration/README.md

```diff
@@ -59,6 +147,8 @@ public static class ChildConfigurationSchema {
   * `rootName` property assigns a _key_ to the root node of the tree that will represent 
     the corresponding configuration schema;
 * `@Config` is similar to the `@ConfigurationRoot` but represents an inner configuration node;
+* `@PolymorphicConfig` is similar to the `@Config` and abstract class in java i.e. cannot be instantiated, but they can be subclassed;
+* `@PolymorphicConfigInstance` marks the inheritor of a polymorphic configuration, has the only property `value`, which is a unique identifier;
```

Comment:
It will be useful to describe how is this identifier going to be used and how unique it actually need to be (e.g. is it universally unique or only among subclasses of a single schema?)

Reply:
Fixed, I guess